promgen/templates/promgen/farm_detail.html

```diff
@@ -4,7 +4,11 @@
 {% block content %}
 
 <div class="page-header" v-pre>
-  <h1>Farm: {{ farm.name }} ({{ farm.source }})</h1>
+  <h1>Farm: {{ farm.name }} ({{ farm.source }})
```

Comment:
```suggestion
  <h1>
    Farm: {{ farm.name }} ({{ farm.source }})
```

If the content of the `<h1>` element is going to be split into multiple lines, do it from the very beginning.

Reply:
Fixed in d176e52.